Clarify `compress_after` parameter (#971)
timescale/docs: api/add_compression_policy.md

|Name|Type|Description|
|---|---|---|
|`hypertable`|REGCLASS|Name of the hypertable or continuous aggregate|

> |`compress_after`|INTERVAL or INTEGER|The age after which the policy job compresses chunks. `compress_after` is calculated relative to the current time. In other words, chunks containing data older than `now - {compress_after}::interval` are compressed.|

Reviewer: ```suggestion
|`compress_after`|INTERVAL or INTEGER|The age after which the policy job compresses chunks. `compress_after` is calculated relative to the current time, so chunks containing data older than `now - {compress_after}::interval` are compressed.|
```
